### ISSUE TITLE
Feed name missing

### ISSUE THREAD

**Issue by wrycu** [jekyll/jekyll-feed#394]
It appears that the feed name is not being picked up by e.g. `inroreader` despite the site having both a name and a title. From the debugging I've done, it may be due to the fact that the name is placed into HTML `div`s, resulting in it being removed during sanitization.  In any case, though, not having the name show up in one of the more popular feed readers is something I figured you may want to know.

**Comment by Jmuccigr** [comment]
FYI, mine is just fine in Inoreader. It even picks up the subtitle. The title gets wrapped in `<title type="html"></title>`. For comparison, NetNewsReader picks your title up just fine.

Seems like a bug in Inoreader. What happens if you put in a subtitle via a description in the config? That's the only difference with mine apart from the version of jekyll.

**Comment by wrycu** [comment]
Thanks for the comment! Can you expand on what you mean by "put in a subtitle via a description"?

**Comment by Jmuccigr** [comment]
> Thanks for the comment! Can you expand on what you mean by "put in a subtitle via a description"?

In your _config.xml, add a line like this:

```yml
description: This is my blog.
```

The should result in a `<subtitle> This is my blog.</subtitle>` getting added to your feed.xml once GitHub processes it, which you can then check for.

Again, I have no idea if this is the issue.

PS I'm picking up this as your favicon from one line in your default.html, even though the other icon references refer to the favicon files:
```html
<link rel="mask-icon" href="/assets/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
```

**Comment by Jmuccigr** [comment]
Any luck?

**Comment by wrycu** [comment]
Nope, is still shows `<div>Leetsaber</div>` when I first follow it and then nothing after in the actual menu.

**Comment by Jmuccigr** [comment]
Yeah, the feed now looks just like mine, so I suspect this is a bug in inoreader. Again, the title shows up just fine in the NewNewsWire app.
